paper/src/main/java/com/untamedears/realisticbiomes/growth/FungusGrower.java

```diff
@@ -46,13 +46,16 @@ public void setStage(final Plant plant, final int stage) {
 				material == Material.CRIMSON_FUNGUS ? TreeFeatures.CRIMSON_FUNGUS :
```

Comment:
Can we please at the same time remove the ternary operator here, and make it something more sensible? This is completely unreadable and opaque code.

Reply:
I suggest separating refactoring of unrelated (to fix) code and refactoring.

Reply:
I mean can we make refactoring as a separate PR.